packages/bpk-theming/src/BpkThemeProvider-test.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import BpkThemeProvider from './BpkThemeProvider';
+
+describe('BpkThemeProvider', () => {
+  it('should render correctly', () => {
+    const tree = renderer.create(
+      <BpkThemeProvider
+        theme={{ color: 'white' }}
+        themeAttributes={['color']}
+      >
+        <p>Lorem Ipsum</p>
+      </BpkThemeProvider>,
+    ).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('should render correctly when theme is missing attributes', () => {
+    console.warn = jest.fn(); // eslint-disable-line no-console
+    const tree = renderer.create(
+      <BpkThemeProvider
+        theme={{}}
+        themeAttributes={['color']}
+      >
+        <p>Lorem Ipsum</p>
+      </BpkThemeProvider>,
+    ).toJSON();
+    expect(tree).toMatchSnapshot();
+    expect(console.warn).toHaveBeenCalledWith('BpkThemeProvider: To apply theming, the theme prop must include `color` (missing `color`)'); // eslint-disable-line max-len,no-console
+  });
+
+  it('should strip out any theme attributes not listed in themeAttributes', () => {
```

Comment:
This is not what this tests right? It should say something like

```
it('should warn if not all `themAttributes` are specified')
```

Reply:
You're right, now updated.